src/Hl7.Fhir.Serialization.Shared.Tests/ParseDemoPatientLineInfoAnnotationPoco.cs

```diff
@@ -0,0 +1,87 @@
+﻿using FluentAssertions;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Utility;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections;
+using System.IO;
+
+namespace Hl7.Fhir.Serialization.Tests
+{
+    [TestClass]
+    public class ParseDemoPatientLineInfoAnnotationPoco
+    {
+        public T getXmlPocoAnnotated<T>(string xml) where T : Base
```

Comment:
public method, so first character should be capitalized!  Are you a Java programmer 🤣 ?